scene/2d/position_2d.h

```diff
@@ -37,14 +37,21 @@ class Position2D : public Node2D {
 
 	GDCLASS(Position2D, Node2D)
 
+	const float default_gizmo_extents = 10.0;
```

Comment:
The constant should be in capital letters, otherwise it looks good to me.

Reply:
I couldn't find other constants to see how it should look. Fixed.